src/main/resources/templates/db_scripts/schema.json

```
@@ -633,6 +644,11 @@
           "fieldName": "illPolicyId",
           "targetTable": "ill_policy",
           "tOps": "ADD"
+        },
+        {
+          "fieldName": "holdingsRecordsSourceId",
```

The property has `sourceId` name in schema 

```suggestion
          "fieldName": "sourceId",
```